Mathlib/Tactic/Linter/TextBased.lean

```diff
@@ -146,8 +146,9 @@ inductive ComparisonResult
 and, if it is, if we prefer replacing the new exception or keeping the previous one. -/
 def compare (existing new : ErrorContext) : ComparisonResult :=
   -- Two comparable error contexts must have the same path.
-  if existing.path != new.path then
-    ComparisonResult.Different
+  -- To avoid issues with different path separators across different operating systems,
+  -- we compare the set of path components instead.
+  if existing.path.components != existing.path.components then ComparisonResult.Different
```

Comment:
```suggestion
  if existing.path.components != new.path.components then ComparisonResult.Different
```
Maybe?  Otherwise the two sides look identical!

Reply:
Oh yes!